projects/packages/forms/src/blocks/field-textarea/edit.js

```diff
@@ -34,26 +33,21 @@ export default function TextareaFieldEdit( props ) {
 	} );
 
 	const defaultLabel = __( 'Message', 'jetpack-forms' );
+
+	const templateLabel = label ?? '';
```

Comment:
Since the label on the text area block is kind of perfunctory (we use the label block's own label attribute after migration), I think we can just get rid of the `useEffect`, and work out the correct label here instead.

I find it a bit unusual though, maybe we can also get rid of this and the block will still work as expected. 🤔

Reply:
This sounds like a good one to punt to a follow-up. 

The useEffect was a remnant from the pre-inner-block edit components. I'm really not a fan of all the form block attributes having empty string defaults however changing that means extra updates needed for the controls to not bounce between being controlled/uncontrolled.

Also, there was a small fix earlier today around how the Label block's placeholder fallbacks were determined. For a new block insertion using this template we might be able to just avoid setting the label at all.